cli/cmd/inject_test.go

```diff
@@ -294,19 +299,22 @@ func testReadFromFolder(t *testing.T, resourceFolder string, expectedFolder stri
 		t.Fatal("Unexpected error. Exit code from runInjectCmd: ", exitCode)
 	}
 
-	expected := readOptionalTestFile(t, filepath.Join(expectedFolder, "injected_nginx_redis.yaml"))
+	expectedFile := filepath.Join(expectedFolder, "injected_nginx_redis.yaml")
+	expected :, expectedFile= readTestdata(t)
```

Comment:
Typo here -- looks like it should be:

```suggestion
	expected := readTestdata(t, expectedFile)
```

But that causes a different compilation error on 305, in the call to `writeTestdataIfUpdate`.

Reply:
i thought i would very cleverly reorder the arguments with one cool vscode command ;( thanks for catching